xml/System.Net/WebProxy.xml

```diff
@@ -376,7 +376,8 @@
   
 ## Examples  
  The following code example demonstrates calling this constructor.  
-  
+ An Example for the Syntax would be nice like why you need ; before * or if you need exclusions for RFC 1918 Networks.
```

Comment:
@PehDeh please open an issue for this recommendation so we can track it. PR-s are for actual documentation changes.